Libs/Reflex/src/main/java/reflex/value/internal/ReflexInternalValue.java

```diff
@@ -27,11 +27,11 @@
 
 public class ReflexInternalValue extends ImmutableReflexValue {
 
-    public ReflexInternalValue(int lineNumber, Object v) {
+    public ReflexInternalValue(int lineNumber, Internal v) {
```

Comment:
Code correctness because the argument to ReflexInternalValue should always be an Internal enum.